src/math/hyp1f1.hh

```diff
@@ -601,21 +632,26 @@ namespace rascal {
         }
       }
 
-      //! computes G using recursion relations
-      //! (derivative are always computed)
+      /**
+       *  Computes G and dG/dz*dz/dr using recursion relations with
+       *  z = (alpha * r_ij)**2 / (alpha + fac_b)
+       */
       inline void calc_recursion(const double & r_ij, const double & alpha,
                                  const Vector_Ref & fac_b) {
         double M1p2p{0.}, M2p3p{0.}, MP1p2p{0.}, MP2p3p{0.}, M1p1p{0.}, Moo{0.},
```

Comment:
this is important to match the tex formulas. since they are complicated enough I would keep the cap for the ease of readability

Reply:
I agree that the M should be left capitalized to fit mathematical convention; doing otherwise would seriously hurt readability. But I still have no idea what `1p2p` means (derivatives?), nor the capital `P`, nor the cow noise at the end of this line - is there anything more descriptive we can use for these meanings?